package.json

```diff
@@ -14,7 +14,7 @@
     "watch": "yarn watch:browser",
     "watch:browser": "cozy-scripts start --browser --hot",
     "watch:mobile": "cozy-scripts watch --mobile",
-    "start": "cozy-scripts start --hot --browser",
+    "start": "yarn watch",
```

Comment:
we should have one commit, one stuff

Reply:
yes, of course, let's merge this first: https://github.com/cozy/cozy-home/pull/1745